chapter02/rmn/ex1.pl

```diff
@@ -0,0 +1,14 @@
+use 5.014;
```

Comment:
Feels incomplete to see no shebang at the start. But then again windows doesn't need one I suppose.